docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner
+
+### Policy Reports
```

Comment:
a doc page for the PolicyReports CRD exist at:

https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/045372e558b896695b2daae92e8c7a04d4d40282/policy-report/docs/index.html

It's quite a mouthful, and notice how I have just pinned to a specific commit that shows the v1alpha2 version. But I think we should use that specific commit.

Upstream has https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html, which tracks `master` and shows docs for incomplete `v1beta1`. They are using that link at https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/README.md.

Triggered https://doc.crds.dev/github.com/kubernetes-sigs/wg-policy-prototypes but doesn't seem to recognize them.